docs/Makefile

```diff
@@ -21,9 +21,9 @@ help:
 
 clean:
 	rm -rf $(BUILDDIR)/*
-	rm -rf auto_examples api cache gen_modules __pycache
+	rm -rf api cache gen_modules __pycache
 
-html-noplot:
-	$(SPHINXBUILD) -D plot_gallery=0 -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+html-nonb:
+	$(SPHINXBUILD) -D nbsphinx_execute='never' -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	@echo
-	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
+	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html. Jupyter notebooks were skipped."
```

Comment:
It would be nice to have a command like this in the `tox` `build_docs` environment as well.  These options should also be mentioned in the documentation itself, probably in the **Documentation Guidelines** section.